src/components/Input.scss

```diff
@@ -158,5 +159,39 @@ $form-input-bg-color: $cu-foreground--dark !default;
       @include ss-pika;
     }
   }
+}
 
+// * 5. Errors and feedback
+.Input-feedback {
+  display: none;
+}
+.is-required.is-invalid {
+  .Input-label:after {
+    color: $cu-negative;
+  }
 }
+.is-required.is-valid {
+  .Input-label:after {
+    content: ' ';
+  }
```

Comment:
What is this doing?

Reply:
It's removing the is-required star from the form if the element is marked with is-valid.

https://teamsnap.invisionapp.com/share/QTBYOK53R#/screens/247877030

The stars aren't there when the form element is valid. Also, the is-required star in the style guide is before the label element. As it exists now, it is built on the :after of the label. Let me know if that is not okay and I can correct that as well.

Reply:
Ah, I see. Yeah, I guess the `:before` `:after` is a tad confusing, but often these aren't used to position things in any order so it doesn't bother me too much. Totally fine if you want to change that, though. Could you just add a line of documentation here to say what the `content: ''` is doing? Since it is out of context from the star placement, it is not obvious.